### DISCUSSION TITLE
Various questions.

### DISCUSSION THREAD

**Discussion by SavantSusi** [mholtkamp/octave#3] [Q&A] [answered]
1. Are you considering on porting it to other consoles? Be nice to see it on other 6th gen consoles, maybe even 7th gen. Vintage PC ports for like Windows 98 or XP would be cool too.

2. What kind of audio does it support? Is it midi or a tracker of sorts can you put in streamed audio? If so what format and specifications?

3. Is video playback supported? If so, format and specs?

4. Can it use the system's internal clock?

5. Can it save to a gamecube memory card or the wii system memory?

**Comment by mholtkamp** [comment, marked as answer]
Hey Susi!

As far as other consoles, the only one I have been thinking about is PS Vita, but that's farther in the future. I will probably get an Android port working before that too.

The only audio formats supported are WAV files, which isn't great but it is simple and versatile.

No video playback is supported.

I don't think you can use the internal clock. I'm not sure how standard library functions work with respect to tthat

You can't save to memory card yet, but it is definitely a feature I'll be adding in the near future. I think I have some functions for generic file read and writes, so maybe that works on the Wii system memory
